apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -86,11 +86,6 @@ export default {
       type: Array,
       required: false,
     },
-    // defaultValue: when creating new record, this is initialization value
```

Comment:
are you sure this is never used? this is a way to set default value via another route then via the metadata.

Reply:
A quick search through the entire application tells me it is not being used anywhere (I looked at where the RowEdit component was being imported).